### PR TITLE
VMO-3679  Builder should only output resource value with single SupportedContentType

### PR DESCRIPTION
**Pull request by poojahpatil90**
Note to reviewers: I could not test the storybook changes on my machine, some issue with rendering the storybook view. I will reach out to @safydy-r once he is back to test it on his machine.

Update: I got storybook working on my machine after cloning the project freshly(suggested by safydy). There are no issues on storybook with the new changes introduced by this PR.

### CODE REVIEW

**Review by safydy-r — CHANGES_REQUESTED** [review]
@poojahpatil90 , it's almost there. I left minor comment above to remove one TODO. Apart from that, as `master` no longer has TS issue, could you pls update from master and see if your current TS error will be gone (normally it should be as Jacob has fixed it) ?
```
ERROR in /Users/rs/Documents/Voto/floip/flow-builder/src/store/flow/utils/importHelpers.ts(2,3):
2:3 Module '"@floip/flow-runner"' has no exported member 'IResourceDefinition'.
    1 | import {
  > 2 |   IResourceDefinition,
      |   ^
    3 |   IContext,
    4 |   IBlock,
    5 | } from '@floip/flow-runner'
ERROR in /Users/rs/Documents/Voto/floip/flow-builder/src/store/flow/utils/importHelpers.ts(16,44):
16:44 Parameter 'value' implicitly has an 'any' type.
    14 | export const updateResourcesForLanguageMatch = function (resources: IResourceDefinition[], oldId: string, newId: string): IResourceDefinition[] {
    15 |   return resources.map((resource) => {
  > 16 |     resource.values = resource.values.map((value) => {
       |                                            ^
    17 |       if (value.languageId === oldId) {
    18 |         value.languageId = newId
    19 |       }
Version: typescript 4.2.4
Time: 9131ms
```

Let me know once you have confirmation it's clean, so I'd be happy to approve the PR

File: stories/story-utils/storeSetup.ts [hunk 1]
```diff
@@ -121,12 +121,9 @@ export class BaseMountedVueClassWithResourceAndMode extends BaseMountedVueClass
 
     // Set values on resource editor
     // TODO: find better way to do this once the resource editor is fully implemented, the goal is to set resources' value correctly. The implementation below is just an
```
[the remark this answers is not in the document]
Comment (safydy-r) [reply stories/story-utils/storeSetup.ts:123]: I'm glad we have this fixed. It's been a while we lug this around sprints :p

[the remark this answers is not in the document]
Comment (poojahpatil90) [reply stories/story-utils/storeSetup.ts]: Done!

**Comment by poojahpatil90** [comment]
> @poojahpatil90 , it's almost there. I left minor comment above to remove one TODO. Apart from that, as `master` no longer has TS issue, could you pls update from master and see if your current TS error will be gone (normally it should be as Jacob has fixed it) ?
> 
> ```
> ERROR in /Users/rs/Documents/Voto/floip/flow-builder/src/store/flow/utils/importHelpers.ts(2,3):
> 2:3 Module '"@floip/flow-runner"' has no exported member 'IResourceDefinition'.
>     1 | import {
>   > 2 |   IResourceDefinition,
>       |   ^
>     3 |   IContext,
>     4 |   IBlock,
>     5 | } from '@floip/flow-runner'
> ERROR in /Users/rs/Documents/Voto/floip/flow-builder/src/store/flow/utils/importHelpers.ts(16,44):
> 16:44 Parameter 'value' implicitly has an 'any' type.
>     14 | export const updateResourcesForLanguageMatch = function (resources: IResourceDefinition[], oldId: string, newId: string): IResourceDefinition[] {
>     15 |   return resources.map((resource) => {
>   > 16 |     resource.values = resource.values.map((value) => {
>        |                                            ^
>     17 |       if (value.languageId === oldId) {
>     18 |         value.languageId = newId
>     19 |       }
> Version: typescript 4.2.4
> Time: 9131ms
> ```
> 
> Let me know once you have confirmation it's clean, so I'd be happy to approve the PR

@safydy-r  Merged master to this branch and attached a screenshot for your reference.

Screen Shot 2021-06-07 at 10 28 39 AM

**Review by safydy-r — APPROVED** [review]
@poojahpatil90 , I'm approving it. Good job

**Pull request merged**